cmd/data-scanner.go

```diff
@@ -676,8 +677,7 @@ func (f *folderScanner) scanFolder(ctx context.Context, folder cachedFolder, int
 				path:           prefix,
 				recursive:      true,
 				reportNotFound: true,
-				minDisks:       len(f.disks), // We want full consistency.
-				// Weird, maybe transient error.
+				minDisks:       f.disksQuorum,
```

Comment:
It seems like wasted cycles to heal unless all disks are available.

Reply:
The main reason is @klauspost if some of them are down and some of them just came up we will never prioritize healing over others. 

Keeping it this way to ensure we prioritize the cases for SLA as we do not know when nodes go offline and come back online.

Reply:
I have seen users who leave servers dead for 1month+